### PR TITLE
Explained XRCompositor behavior for additional displays

### LINKED ISSUE #155: Presenting stereo layers to devices with more than two views
https://immersive-web.github.io/layers/#xrcompositorchanges discusses how to present stereo layers when there's only one view, but not when there's more than two (e.g. devices which record to a third view).

### PR DESCRIPTION
**Pull request by cabanier**
Closes #155

### CODE REVIEW

File: webxrlayers-1.bs [hunk 1]
```diff
@@ -1446,6 +1446,10 @@ If the [=XR Compositor=] is driving a monoscopic display and the rendered {{XRCo
 If the {{XRCompositionLayer}} was created through {{XRMediaBinding}} and the {{XRMediaLayerInit/"invertStereo"}} flag was set to
 <code>true</code>, the bottom portion of the {{XRLayer}} is shown while the top portion is ignored.
 
+If the [=XR Compositor=] is driving a additional display in addition to a monoscopic or stereoscopic one, it is up to the
```
Comment (asajeffrey) [inline webxrlayers-1.bs:1449]: Possibly more than one additional displays?

File: webxrlayers-1.bs [hunk 2]
```diff
@@ -1446,6 +1446,10 @@ If the [=XR Compositor=] is driving a monoscopic display and the rendered {{XRCo
...
+user agent how a {{XRCompositionLayer}} with an {{XRLayerLayout}} of {{XRLayerLayout/"stereo-left-right"}} or
+{{XRLayerLayout/"stereo-top-bottom"}} is shown to that additional display.
```
Comment (asajeffrey) [inline webxrlayers-1.bs:1451]: Do we need language about `mono` too?

File: webxrlayers-1.bs (hunk 1 above)
Comment (cabanier) [reply webxrlayers-1.bs]: Fixed with latest PR.

File: webxrlayers-1.bs (hunk 2 above)
Comment (cabanier) [reply webxrlayers-1.bs]: Isn't that already covered by the 2 previous paragraphs?

Comment (asajeffrey) [reply webxrlayers-1.bs]: is it, I don't see any text about how to render mono layers. I mean it's kind of obvious, so maybe it doesn't need spelled out?

Comment (cabanier) [reply webxrlayers-1.bs]: Yes, I don't think it's needed.

**Pull request merged**